repos/fdbt-site/src/pages/defineCapStart.tsx

```diff
@@ -0,0 +1,106 @@
+import TwoThirdsLayout from '../layout/Layout';
+import { getCsrfToken, sentenceCaseString } from '../utils';
+
+const title = 'Define cap start - Create Fares Data Service';
+const description = 'Capped products start selection page of the Create Fares Data Service';
+
+interface SelectCappedProductGroupProps {
+    errors: ErrorInfo[];
+    csrfToken: string;
+}
+
+const daysOfWeek = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
+
+const DefineCapStart = ({ errors = [], csrfToken }: SelectCappedProductGroupProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description} errors={errors}>
+        <CsrfForm action="/api/defineCapStart" method="post" csrfToken={csrfToken}>
+            <ErrorSummary errors={errors} />
+            <div className="govuk-form-group">
+                <fieldset className="govuk-fieldset" aria-describedby="fixed-weekdays-hint">
+                    <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                        <h1 className="govuk-fieldset__heading">Define when cap is calculated from</h1>
+                    </legend>
+                    <h2 className="govuk-heading-m govuk-!-margin-top-6">Business week</h2>
+                    <span id="fixed-weekdays-hint" className="govuk-hint">
+                        Which day does your cap&apos;s calculation start on?
+                    </span>
+                    <div className="govuk-radios" data-module="govuk-radios">
+                        <div className="govuk-radios__item">
+                            <input
+                                className="govuk-radios__input"
+                                id="fixed-weekdays"
+                                name="capStart"
+                                type="radio"
+                                value="fixedWeekdays"
+                                data-aria-controls="conditional-fixed-weekdays"
+                            />
+                            <label className="govuk-label govuk-radios__label" htmlFor="fixed-weekdays">
+                                Fixed weekdays
+                            </label>
+                        </div>
+                        <div
+                            className="govuk-radios__conditional govuk-radios__conditional--hidden"
+                            id="conditional-fixed-weekdays"
+                        >
+                            <div className="govuk-form-group">
+                                <label className="govuk-label" htmlFor="start">
+                                    Start
+                                </label>
+                                <select className="govuk-select" id="start" name="start">
+                                    {daysOfWeek.map((day) => (
+                                        <option defaultValue={day[0]} key={day} value={day}>
+                                            {sentenceCaseString(day)}
```

Comment:
upperFirst from lodash is what we want here (I know sentenceCaseString will work, but upperFirst is more fit for purpose).